src/main/java/io/netty/incubator/channel/uring/IOUringEventLoop.java

```diff
@@ -54,6 +57,9 @@
     private final byte[] inet4AddressArray = new byte[SockaddrIn.IPV4_ADDRESS_LENGTH];
     private final byte[] inet6AddressArray = new byte[SockaddrIn.IPV6_ADDRESS_LENGTH];
 
+    private final IOUringCompletionQueueCallback callback = IOUringEventLoop.this::handle;
```

Comment:
This because now that the class is public, you don't want to expose the interface?

Reply:
yes.